src/components/ActorsTodo/ActorsTodo.tsx

```diff
@@ -0,0 +1,69 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+import * as S from "./ActorsTodo.Styled";
+import { useState } from "react";
+
+interface TypesForActors {
+	id: number;
+	name: string;
+}
+
+function ActorsTodo() {
+	const [actorsList, setActorsList] = useState<TypesForActors[]>([]);
+	const [newActor, setNewActor] = useState("");
+
+	const addActor = () => {
+		if (newActor.trim() !== "") {
+			const actorsObject = {
```

Comment:
`actor` object, zwróć uwagę w przyszłości na odmianę, liczbę poj/mnogą, to też ma znaczenie w nazwach.

Dodajesz aktora, więc actorObject, lub po prostu `payload` to taka nazwa na dane wysyłane gdzieś do api/stanu